<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterOptionsUI</class>
 <widget class="QWidget" name="FilterOptionsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>404</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="kcfg_webShortcutsEnabled">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable or disable web search keywords. &lt;/p&gt;&lt;p&gt;Web search keywords allow you to quickly access or search for information located online or on your hard drive. &lt;/p&gt;&lt;p&gt;KDE comes with many predefined keywords. One such keyword can be used to perform a the Google (TM) search; to use it, you simply type the keyword 'gg' followed by the keyword delimiter and the search term, e.g. &lt;span style=&quot; font-weight:600;&quot;&gt;gg:KDE&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Enable Web search keywords</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="kcfg_favoritesOnly">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Use preferred keywords only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="placeholderText">
      <string>Search for web search keywords</string>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTreeView" name="lvSearchProviders">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbNew">
       <property name="toolTip">
        <string>Add a new Web search keyword</string>
       </property>
       <property name="text">
        <string>&amp;New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbChange">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Modify the highlighted Web search keyword</string>
       </property>
       <property name="text">
        <string>Chan&amp;ge...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delete the highlighted Web search keyword</string>
       </property>
       <property name="text">
        <string>De&amp;lete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>158</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbDefaultEngine">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>&lt;qt&gt;
Select the search engine to use for input boxes that provide automatic lookup services when you type in normal words and phrases instead of a URL. To disable this feature select &lt;b&gt;None&lt;/b&gt; from the list.
&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string>Default Web &amp;search keyword:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>cmbDefaultEngine</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbDefaultEngine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>67</width>
         <height>0</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select a default web search keyword. &lt;/p&gt;&lt;p&gt;This allows applications to automatically convert the typed word or phrase to web search keyword queries when they cannot be filtered into a proper URL. &lt;/p&gt;&lt;p&gt;To disable this functionality select &lt;span style=&quot; font-weight:600;&quot;&gt;None&lt;/span&gt; from the list. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbDelimiter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Choose the delimiter that separates the keyword from the phrase or word to be searched.</string>
       </property>
       <property name="text">
        <string>&amp;Keyword delimiter:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>cmbDelimiter</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbDelimiter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Choose a delimiter to mark the Web search keyword.</string>
       </property>
       <item>
        <property name="text">
         <string comment="Colon as keyword delimiter">Colon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="Space as keyword delimiter">Space</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_webShortcutsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_favoritesOnly</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_webShortcutsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>lvSearchProviders</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_webShortcutsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbNew</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
